packages/web-components/fast-foundation/src/menu/menu.ts

```diff
@@ -249,7 +249,7 @@ export class Menu extends FoundationElement {
             item.addEventListener("expanded-change", this.handleExpandedChanged);
             item.addEventListener("focus", this.handleItemFocus);
 
-            if (item instanceof MenuItem) {
+            if ("startColumnCount" in item) {
```

Comment:
Just a note that this API has been removed for vNext in favor of folks extending and implementing themselves, so any changes here will not work beyond the current stable versions.

Thoughts on simply adding to the current check? 

```suggestion
            if (item instanceof MenuItem || "startColumnCount" in item) {
```

Reply:
Thanks for the heads up. And I'm fine with just adding to the check.